tests/utils.sh

```diff
@@ -376,18 +376,25 @@ function assert_no_line_match()
 # @result_to_compare Raw output to be compared
 function assert_equals_helper()
 {
+  # colors to use to print error
+  local _none='\033[0m'
+  local _red='\033[1;31m'
+  local _green='\033[1;32m'
```

Comment:
I forgot to ask, but why add `_`? We usually use it when dealing with references (i.e., `local -n _variable`). If there is no particular reason, please, rename these variables to not use `_`.

Reply:
I myself was going to rename them later, just forgot. Corrected.